hedera-node/hedera-mono-service/src/main/java/com/hedera/services/txns/crypto/AutoCreationLogic.java

```diff
@@ -195,41 +197,54 @@ public Pair<ResponseCodeEnum, Long> create(
                     "Cannot auto-create an account from unaliased change " + change);
         }
 
-        // checks tokenAliasMap if the change consists an alias that is already used in previous
-        // iteration of the token transfer list. This map is used to count number of
-        // maxAutoAssociations needed on auto created account
-        analyzeTokenTransferCreations(changes);
+        final var isAliasEVMAddress = alias.size() == EntityIdUtils.EVM_ADDRESS_SIZE;
+        if (isAliasEVMAddress && !properties.isLazyCreationEnabled()) {
+            return Pair.of(NOT_SUPPORTED, 0L);
```

Comment:
Since we have already checked the validity in `ImpliedTransfersMarshal#unmarshalFromGrpc()` and the transaction will fail before we reach this, maybe we can delete the check here?